src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Blueprint</PackageId>
+    <Description>Provides Microsoft Azure Blueprint management operations including the ability to create, update and delete Blueprint definition and assignment.</Description>
+    <Version>0.1.0-Preview</Version>
+    <!-- Remove the Preview description in PackageReleaseNotes once the preview is removed.-->
+    <AssemblyTitle>Microsoft Azure Blueprint Management Library</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.Blueprint</AssemblyName>
+    <PackageTags>Blueprint;</PackageTags>
+    <PackageReleaseNotes>
```

Comment:
Please add some relevant information in `PackageReleaseNotes`
An example [here](https://github.com/Azure/azure-sdk-for-net/blob/53b16a9d7954076f9002013846c4c2840cebf92b/src/SDKs/DevSpaces/Management.DevSpaces/Microsoft.Azure.Management.DevSpaces.csproj#L12)

Reply:
Please add `AssemblyInfo.cs` similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Logic/Management.Logic/Properties/AssemblyInfo.cs) with versioning info

Reply:
✅

Reply:
✅